app/services/notices/setup/abstraction-alerts/submit-alert-email-address.service.js

```diff
@@ -50,6 +51,24 @@ async function _save(session, payload, auth) {
   return session.$update()
 }
 
+/**
+ * Combines the existing session data with the submitted payload formatted by the presenter. If nothing is submitted by
+ * the user, payload will be an empty object.
+ *
+ * @private
+ */
+function _submittedSessionData(session, auth, validationResult, payload) {
```

Comment:
As far as i am aware we don't do this anywhere else. 

Why can't this be done in the presenter ?